[TT-7347] Update Helm Chart docs for tyk-helm-chart release v0.13.2 (#2185)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-self-managed/tyk-helm-chart.md

Tyk Self-Managed licensing allow for different numbers of Gateway nodes to connect to a single Dashboard instance.

> To ensure that your Gateway pods will not scale beyond your license allowance, set the Gateway's resource kind to *Deployment*

Reviewer: Gateway resource is already set to Deployment. users do not need to take any further actions to modify resource kind. Maybe instead of `set the Gateway's resource kind to *Deployment*`, we can say something like `please ensure that the Gateway's resource kind is *Deployment*`

> To ensure that your Gateway pods will not scale beyond your license allowance, please ensure that the Gateway's resource kind is *Deployment*

Reviewer 2: ```suggestion
To ensure that your Gateway pods will not scale beyond your license allowance, please ensure that the Gateway's resource kind is `Deployment`
```

> and the replica count to your license node limit. By default, the chart is configured to work with single node license: `gateway.kind=Deployment` and `gateway.replicaCount=1` in `values.yaml`. You can update the default settings using following options: `--set gateway.kind=Deployment --set gateway.replicaCount=1` in your `values.yaml` file or in the Helm install command.

Reviewer: ```suggestion
and the replica count to your license node limit. By default, the chart is configured to work with a single node license: `gateway.kind=Deployment` and `gateway.replicaCount=1`.
```
